src/main/java/calculator/StringParser.java

```diff
@@ -6,19 +6,22 @@
 public class StringParser {
 
     private static final Pattern CUSTOM_DELIMITER_PATTERN = Pattern.compile("//(.)\\\\n(.*)");
+    private static final int CUSTOM_DELIMITER_INDEX = 1;
+    private static final int ONLY_NUMBER_STRING_INDEX = 2;
+    private static final String DEFAULT_DELIMITERS = ",|:";
 
     private StringParser() {
 
     }
 
     public static String[] splitNumbers(String text) {
         Matcher m = CUSTOM_DELIMITER_PATTERN.matcher(text);
-        String delimiters = ",|:";
+        String delimiters = DEFAULT_DELIMITERS;
 
         if (m.find()) {
-            String customDelimiter = m.group(1);
+            String customDelimiter = m.group(CUSTOM_DELIMITER_INDEX);
             delimiters = delimiters + "|" + customDelimiter;
-            text = m.group(2);
+            text = m.group(ONLY_NUMBER_STRING_INDEX);
         }
 
         return text.split(delimiters);
```

Comment:
상수로 의미부여 👍 

저의 경우 얼리리턴하는 것을 선호해요.
얼리 리턴의 장점에 대해서도 고민해보시면 좋을 것 같아요 😄 

```suggestion
        final Matcher m = CUSTOM_DELIMITER_PATTERN.matcher(text);
        if (!m.find()) {
            return text.split(DEFAULT_DELIMITERS);
        }
        
        final String customDelimiter = m.group(CUSTOM_DELIMITER_INDEX);
        final String delimiter = String.join("|", DEFAULT_DELIMITERS, customDelimiter);

        final String numberString = m.group(ONLY_NUMBER_STRING_INDEX);
        return numberString.split(delimiter);
```